### PR TITLE
Fix ToC scrolling

### LINKED ISSUE #13: ToC does not scroll correctly
image

### PR DESCRIPTION
**Pull request by GeekPro101**
Fixes #13 
Note this also makes the py-10 change

### CODE REVIEW

**Comment by apple-phi** [comment]
Bit unsure on exactly what this changes. It seems to misalign the ToC, while also not fixing the incomplete scrolling (see img)
image

**Comment by apple-phi** [comment]
Also would be helpful if you could link the issue this resolves too please!

**Comment by GeekPro101** [comment]
image

**Comment by GeekPro101** [comment]
[Screencast_20240417_113823.webm](https://github.com/dowjcr/web-frontend/assets/14115426/1872397f-314e-4995-b326-54736a958859)

**Comment by apple-phi** [comment]
Doesn't seem to have any significant difference in behaviour on my machine.

https://github.com/dowjcr/web-frontend/assets/65281038/de88d144-26ee-4f54-bdb5-e501c5330bc8

**Comment by GeekPro101** [comment]
> Doesn't seem to have any significant difference in behaviour on my machine.
> Recording.2024-04-17.150341.mp4

@apple-phi That'll be because your screen is of sufficient size to display the whole contents without it overlapping

**Comment by apple-phi** [comment]
> @apple-phi That'll be because your screen is of sufficient size to display the whole contents without it overlapping

Isn't this the case for yours also? The whole ToC is shown the whole time. Or am I missing something?

**Comment by GeekPro101** [comment]
> > @apple-phi That'll be because your screen is of sufficient size to display the whole contents without it overlapping
> 
> Isn't this the case for yours also? The whole ToC is shown the whole time. Or am I missing something?

@apple-phi - yes, but a) that might not be the case for a longer ToC and b) see the second video - because its fixed, it will extend over the footer, rather than staying within the page area

File: src/routes/[nav]/[...content]/+page.svelte [hunk 1]
```diff
@@ -53,7 +53,7 @@
 		</article>
 		<div>
 			<aside
-				class="bg-transparent mr-auto h-min flex-none fixed top-20 hidden lg:block xl:px-5 py-20 w-80"
+				class="bg-transparent mr-auto h-min flex-none sticky top-0 hidden lg:block xl:px-5 py-10 w-80"
```
Comment (apple-phi) [inline src/routes/[nav]/[...content]/+page.svelte:56]: Ok I'm happy with this now, we can revisit the spacing at a later point

**Review by apple-phi — APPROVED** [review]

**Pull request merged**